salesagility/SuiteCRM
# Fix # - KB and KB categories metadata changes

## Description




Slight improvements in UI (Knowledge base & KB - Categories)

## Motivation and Context


## How To Test This

1) KB module -> Create KB (edit if you have one) -> check layout for revision field: 
![image](https://github.com/salesagility/SuiteCRM/assets/116086008/736b56d2-b8dd-40c5-acb6-a527aa489eb4)

2) KB categries -> Create one and then Edit: 
2.1) I suggest don't give opportunity to change an Created By;
2.2) No need to show Date Created as it's out of concept of Edit View.

![image](https://github.com/salesagility/SuiteCRM/assets/116086008/3d73550e-f04c-46f1-8860-d926470bb4b0)
   

## Types of changes


### Final checklist
